scripts/main.js

```diff
@@ -0,0 +1,41 @@
+/*----------  Imports  ----------*/
+import * as m from './morse.js'
+import * as du from './domUtils.js'
+
+
+/*----------  Function Declaration  ----------*/
+
+const grabInput = (event) => {
+    // 1. Grab both the input textarea value and the two radio buttons
+    // 2. Check which radio button is selected
+    // 3. Send the textarea value to the associated translation function
+    // 4. Set output textarea value to the return value of the translation function
+
+
+    let inputText = input.value;
+    
+    if (englishToMorseCodeSelect.checked === true) { 
+        console.log(`${inputText} will be translated to Morse Code`);
+        output.value = m.translateEToMC(inputText);
+    } else if (morseCodeToEnglishSelect.checked === true) {
+        console.log(`${inputText} will be translated to English`);
+        output.value = m.translateMCToE(inputText);
+    }
+}
```

Comment:
Because this is a radio button probably fine to check only one is checked otherwise assume the other is. 
- Alternatively assign a value to each radio button and check that value from the form. Not a huge deal either way tbh